in_toto/models/layout.py

```diff
@@ -248,6 +269,38 @@ def __init__(self, **kwargs):
 
     self.validate()
 
+  def __repr__(self):
```

Comment:
Please don't overwrite `Step.__repr__` (or `Inspection.__repr__`). Just give them a display method like you did for Layout and Link.

Reply:
Thank you for this. The Step and Inspection classes both have a display method like Link's and Layout instead of repr methods.